Disable table hover in .content #1295

admench opened the pull request:
This is about **Bulma**.

<!-- Is it a bug/feature/question or do you need help? -->
<!-- If it's a bug, is it a browser bug? -->

### Overview of the problem

<!-- UNCOMMENT THE APPROPRIATE LINES -->

This is about the Bulma **CSS framework**
I'm using Bulma **version** [0.6.0] 

### Description

Referencing #1236 - this issue of the table row `:hover` behaviour is also set as default to `.table` inside `.content` class elements. I propose that we remove this behaviour as default, inline with #1236 as it caught me out after upgrading bulma just to avoid this. 

### Expected behavior

When hovering on a table, by default nothing should happen, only if `.is-hoverable` is added should the visual state change.

### Actual behavior

When interacting with a `.table` within `.content` the hover state is applied by default.

jgthms commented:
Thanks! Can you remove the `$content-table-row-hover-background-color` from the `.sass` file and the docs, and update the changelog?

admench commented:
Absolutely. Wow this is exciting. My first EVER PR!

admench commented:
I have done as requested and removed the unnecessary variable in the sass file and updated the docs to remove the variable mentioned. Does that cover it for the docs?

jgthms commented:
Just need to rebase.

admench commented:
Is there anything that I can do to help?

lmbrochu commented:
@admench you're the one supposed to rebase.

andrewsosa commented:
Since this PR is dead, I've opened a duplicate as #1707, also updating the new docs format.
